### PR TITLE
Fix edit degreeCurricularPlan in coordinator menu ACDM-771 #resolve

### PR DESCRIPTION
**Pull request by pedrorijo91**

### CODE REVIEW

File: src/main/webapp/coordinator/degreeSite/viewDescriptionCurricularPlan.jsp [hunk 1]
```diff
@@ -40,14 +42,11 @@
...
-		<fr:edit name="degreeCurricularPlan" type="org.fenixedu.academic.domain.DegreeCurricularPlan" schema="degree.curricular.plan.description" nested="true">
-		    <fr:layout>
-	    	    <fr:property name="classes" value="tstyle2 thlight thright thtop mbottom2"/>
-	        	<fr:property name="columnClasses" value="width10em,pbottom1,tdclear tderror1 valigntop"/>
-		    </fr:layout>
-		</fr:edit>
-	
+
+		<textarea bennu-html-editor bennu-localized-string name="degreeCurricularPlanDescription" id="degreeCurricularPlanDescription">
+			${degreeCurricularPlanDescription.json()}
```
Comment (jcarvalho) [inline src/main/webapp/coordinator/degreeSite/viewDescriptionCurricularPlan.jsp]: Due to encoding issues, and the fact that inside the JSON can be valid HTML, you must escape the JSON when showing it in the textarea, similar to what is done [here](https://github.com/FenixEdu/fenixedu-cms/blob/v1.0.7/src/main/webapp/WEB-INF/fenixedu-cms/editPost.jsp#L86-L87).

File: src/main/java/org/fenixedu/academic/ui/struts/action/coordinator/DegreeSiteManagementDispatchAction.java [hunk 2]
```diff
@@ -137,16 +144,38 @@ public ActionForward viewDescriptionCurricularPlan(ActionMapping mapping, Action
             return new ActionForward(mapping.getInput());
         }
         request.setAttribute("degreeCurricularPlan", degreeCurricularPlan);
+        request.setAttribute(
+                "degreeCurricularPlanDescription",
+                new LocalizedString().with(Locale.ENGLISH, degreeCurricularPlan.getDescriptionEn()).with(new Locale("pt", "PT"),
```
Comment (pedrosan7os) [inline src/main/java/org/fenixedu/academic/ui/struts/action/coordinator/DegreeSiteManagementDispatchAction.java]: You can use MultiLanguageString.en and MultiLanguageString.pt to access these 'well known' locales.

Comment (NunoPinheiro) [inline src/main/java/org/fenixedu/academic/ui/struts/action/coordinator/DegreeSiteManagementDispatchAction.java]: I would suggest using Locale.ENGLISH and Locale.getDefault() to drop the dependency of "Portuguese"

Comment (jcarvalho) [inline src/main/java/org/fenixedu/academic/ui/struts/action/coordinator/DegreeSiteManagementDispatchAction.java]: And who ensures that the default locale is not English? :)

The actual dependency is in the DegreeCurricularPlan, not here.

**Pull request merged**